mac_pw_pool/setup.sh

```diff
@@ -158,7 +175,7 @@ if ! arch -arch x86_64 /usr/bin/uname -m; then
 fi
 
 msg "Restricting appstore/software install to admin-only"
-if [[ -x /usr/sbin/softwareupdate ]]; then
+if [[ ! -L /usr/local/bin/softwareupdate ]]; then
```

Comment:
Taking a step back, is it really worth preserving the `if`? You're doing `ln -sf` (force) below, so even if this runs twice it should be ok. And it would really simplify life for a maintainer: no need to think about the conditional.

Reply:
Good point.  I can change this easy enough.  It's such a toungue-in-cheek "fix" anyway, best keep it as simple as possible.

Reply:
Investigated doing this, it's not worth it.  The setup script needs to be able to run more than once on a mac.  This symlink's existence "protecting" execution of a few apple-specific config. commands where retrieving their current state is complex.  I could use a separate marker-file, but didn't think there was much point.

I'll add a comment detailing this for future maintainers.